### PR TITLE
Theme Modifications Back-port

### PR DESCRIPTION
**Pull request by mikeselander**
Adding in a theme header and generic bold style as requested by @angtheod.

### CODE REVIEW

File: src/scss/partials/_typography.scss [hunk 1]
```diff
@@ -68,3 +68,7 @@ h2 {
 .white-text {
   color: $white;
 }
+
+.bold {
+  font-weight: bold;
```
Comment (Kluny) [inline src/scss/partials/_typography.scss]: What's the reason for making a .bold class instead of simply using <em>?

Comment (mikeselander) [reply src/scss/partials/_typography.scss]: I don't know - this was requested by @angtheod and I didn't see a particular issue with it.

Comment (kadamwhite) [reply src/scss/partials/_typography.scss]: I'm curious about this too -- `<strong>` seems another potential candidate

**Comment by angtheod** [comment]
Hello @mikeselander ,
not really any reason behind that. Please, feel free to change anything as you prefer.

**Comment by angtheod** [comment]
@mikeselander actually, initially I added a class for it, because I wanted to add this class to the description of those checkboxes that are selected https://github.com/greenpeace/planet4-master-theme/blob/develop/templates/search.twig#L75

So, code looks kind of cleaner to me, if we echo a class using a ternary operator instead of doing that for an html element like <strong>. But its up to you guys to decide. I am fine either way, I will just have to edit those lines in the template.

**Comment by mikeselander** [comment]
I think in that particular case we should be controlling those results in the CSS. The content is the same and it's a state indicator, not an indication of content that's more important than any others. I'll  remove this `bold` class and work that particular use case into the Search styles so that you don't even need that ternary.

**Comment by mikeselander** [comment]
@angtheod that label was bolded automatically in e6ccd0df5aee096ba48a96fb5d5265ca3242ab3b so you can remove that ternary and class now.

**Pull request merged**